ValvePak/ValvePak/Package.Save.cs

```diff
@@ -168,15 +174,20 @@ public void Write(Stream stream)
 			writer.Write(0); // TreeSize, to be updated later
 			writer.Write(0); // FileDataSectionSize, to be updated later
 			writer.Write(0); // ArchiveMD5SectionSize
-			writer.Write(48); // OtherMD5SectionSize
+			writer.Write(48); //OtherMD5SectionSize
 			writer.Write(0); // SignatureSectionSize
-
 			var headerSize = (int)(stream.Position - streamOffset);
-			uint fileOffset = 0;
-
 			const byte NullByte = 0;
 
-			// File tree data
+			bool isSingleFile = entries.Sum(s => s.TotalLength) + headerSize + 64 <= maxFileBytes;
```

Comment:
I don't like using maxFileBytes here, we should just have a bool to specify that we want to multi chunk.

This size calculation is also gonna be incorrect if we want to write file hashes.